### PR TITLE
README.md file

### PR DESCRIPTION
**Pull request by artemklishch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,50 @@
-# bookstore
+### Bookstore application
```
Comment (sarakhmen) [inline README.md]: why do you have three # for this header here and two for other ones? The more hashes, the smaller the header level

File: README.md [hunk 2]
```diff
@@ -1,13 +1,50 @@
...
+3. CartController. This controller is responsible for CRUD operations with cart.
+4. CategoryController. This controller handles CRUD operations with book categories.
+5. OrderController. Responsible for CRUD operations with book orders.
+
+## Launch the project locally
 
 Running the server for the first time:
-1. Clone the project
+1. Clone the project from the GitHub repository: https://github.com/artemklishch/bookstore
 2. Remove './target' directory if it exists with the command 'rm -rf target' (for the first running the server)
```
Comment (sarakhmen) [inline README.md]: this step is redundant as mvn clean automatically removes it

File: README.md [hunk 3]
```diff
@@ -1,13 +1,50 @@
...
 4. Add ".env" file in the root of the project with values for variables as it is pointed in the ".env.example" file - if you didn't add it earlier
+You can use your values of variables in the '.env' file 
 5. Run the server container with the command "docker compose up --force-recreate --build"
 
 Running the server next:
 1. If you don't have "target" directory run the command "mvn clean package"
 2. If you don't have the ".env" file - add it with values for variables as it is pointed in the ".env.example" file
 3. Run the server container with the command "docker compose up --force-recreate --build"
+
+Video instruction to run the app: https://www.loom.com/share/f32fb6dfe9624ae58157ee6732655667?sid=33b2152a-1f9e-49e9-a7a4-a5a36c0ddf12
```
Comment (sarakhmen) [inline README.md]: Can the URL be hidden on the readme? You can embed the video but you don't need to display it separately. Please review the preview of your readme and fix formatting/styling issues

File: src/main/resources/application.properties [hunk 4]
```diff
@@ -8,5 +8,5 @@ spring.jpa.show-sql=true
 spring.jpa.open-in-view=false
 server.servlet.context-path=/api
 
-jwt.expiration=300000000000
+jwt.expiration=3000000
```
Comment (sarakhmen) [inline src/main/resources/application.properties]: this PR shouldn't contain this change

**Review by sarakhmen — CHANGES_REQUESTED** [review]
let's improve your solution. Please check your grammar with google translate, gramarly or other services and fix issues

File: src/main/resources/application.properties (hunk 4 above)
Comment (artemklishch) [reply src/main/resources/application.properties]: ++, reverted

File: README.md (hunk 2 above)
Comment (artemklishch) [reply README.md]: ++, updated

File: README.md (hunk 1 above)
Comment (artemklishch) [reply README.md]: ++, updated

File: README.md (hunk 3 above)
Comment (artemklishch) [reply README.md]: ++, updated

Comment (artemklishch) [reply README.md]: ++, updated

File: README.md [hunk 5]
```diff
@@ -1,13 +1,49 @@
...
+## Controllers
+There are controllers:
+1. AuthenticationController. 
+This controller provides the way of registration of user and signing in of registered user.
+2. BookController. This controller handles CRUD operations with books.
+3. CartController. This controller is responsible for CRUD operations with cart.
+4. CategoryController. This controller handles CRUD operations with book categories.
+5. OrderController. Responsible for CRUD operations with book orders.
+
+## Launch the project locally
```
Comment (Elena-Bruyako) [inline README.md:31]: It's better to add something about initial set up like jdk, docker

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
see comment

Aslo, you can add Models and relations diagram

File: README.md (hunk 5 above)
Comment (artemklishch) [reply README.md:31]: ++, updated

**Review by Elena-Bruyako — APPROVED** [review]
well done

**Pull request merged**